### PR TITLE
FEATURE: TMA-1076 support new version api 2

### PR DESCRIPTION
**Pull request by tqtu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/230/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/231/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/232/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/233/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/234/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/235/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/236/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/237/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/238/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/239/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/240/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/241/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/242/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/243/ : SUCCESS in 4s

File: spec/integration/models/project_spec.rb [hunk 1]
```diff
@@ -11,20 +11,66 @@
...
   after(:all) do
     @client.disconnect
   end
 
   describe 'projects' do
-    it 'Can get all projects' do
-      projects = @client.projects(:all, 100)
+    it 'Can get all projects and not limit' do
+      projects = @client.projects(:all)
+      expect(projects).to_not be_nil
```
Comment (danh-ung) [inline spec/integration/models/project_spec.rb]: can we check if it returns all projects instead of one page.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/244/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/245/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/256/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/257/ : SUCCESS in 16s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/258/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/259/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/260/ : SUCCESS in 7s

**Comment by danh-ung** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/261/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/263/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/264/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/266/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/267/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/268/ : SUCCESS in 8s

**Comment by tqtu** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/272/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/275/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/279/ : SUCCESS in 0s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/281/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/283/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/284/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/285/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/286/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/287/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/288/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/289/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/291/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/292/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/293/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/294/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/295/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/296/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/297/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/422/ : SUCCESS in 8s

**Pull request merged**